astropy/time/core.py

```diff
@@ -112,6 +113,23 @@
         'IAU1994': {'function': erfa_time.gst94, 'scales': ('ut1',)}}}
 
 
+class TimeInfo(BaseInfo):
+    """
+    Make dtype and unit be None and be read-only.  Setting attrs_from_parent
+    to these read-only values is needed so they don't get copied.
+    """
+    attrs_from_parent = set(['unit'])
+
+    @property
+    def unit(self):
+        return None
+
+    info_summary_stats = staticmethod(
+        data_info_factory(names=BaseInfo._stats,
```

Comment:
Does this work currently? in particular, it seems one should not be able to get a `mean` yet:
```
np.mean(Time(['J2000.', 'J2001']))
# OperandTypeError: Unsupported operand type(s) for +: 'Time' and 'Time'
```